src/crimson/os/cyan_object.h

```diff
@@ -16,7 +16,7 @@ struct Object : public boost::intrusive_ref_counter<
   using bufferlist = ceph::bufferlist;
 
   bufferlist data;
-  std::map<std::string,bufferptr> xattr;
+  std::map<std::string,bufferptr,std::less<>> xattr;
```

Comment:
Oh, I see what is going here. This might be worth a comment referencing e.g. https://en.cppreference.com/w/cpp/utility/functional/less_void.

Reply:
added and repushed.